accounting_reporting_core/src/main/java/org/cardanofoundation/lob/app/accounting_reporting_core/resource/views/DataSourceView.java

```diff
@@ -0,0 +1,13 @@
+package org.cardanofoundation.lob.app.accounting_reporting_core.resource.views;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+
+public enum DataSourceView {
+
+    NETSUITE,
+
+    ANOTHER;
```

Comment:
Just remove ANOTHER, we will have NetSuite for now.

This whole topic requires also own discussion and design / planning via our backend connectors but for now we can hard-code this.